src/sdk/pynni/nni/hyperband_advisor/hyperband_advisor.py

```diff
@@ -100,7 +101,7 @@ def json2parameter(ss_spec, random_state):
                 _index = random_state.randint(len(_value))
                 chosen_params = json2parameter(ss_spec[NodeType.VALUE][_index], random_state)
             else:
-                chosen_params = eval('parameter_expressions.' + # pylint: disable=eval-used
+                chosen_params = eval('parameter_expressions.' +  # pylint: disable=eval-used
```

Comment:
Why do we use `eval`???

Reply:
I don't know. Refactoring Hyperband is beyond the responsibility of this PR. What's your suggestion?

Reply:
Seems what it's doing is `getattr(parameter_expressions, _type)`

Reply:
Didn't know `getattr` works with module. (I checked. It works.)

Searched the repo. There seems a lot of `eval`. Will refactor in #1570.